games-fps/prboom-plus/prboom-plus-2.5.1.4.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools desktop xdg
+
+DESCRIPTION="A Doom source port developed from the original PrBoom project"
+HOMEPAGE="http://prboom-plus.sourceforge.net"
+# We are using a github mirror here because the upstream tarball is missing the
+# free dog assets; we should update SRC_URI to point to the upstream tarball
+# once the free assets are present as expected
+#SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+SRC_URI="https://github.com/coelckers/prboom-plus/archive/f96f891d068dcc5ec52ed91056b46d27e9a8462d.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+
+	nonfree? ( freedist )
+	!nonfree? ( BSD )"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="dumb fluidsynth +mad net nonfree +opengl pcre +png portmidi sdl-image +sdl-mixer +vorbis"
+REQUIRED_USE="sdl-image? ( opengl )"
+
+DEPEND="
+	media-libs/libsdl
```

Comment:
I'm wondering whether this should have some USE flags. I'm not sure whether sound works when SDL_mixer is disabled, possibly not. Judging by the code, it should at least have `[opengl?,video]`. I find that `[threads]` is often needed too.

More importantly, all your SDL dependencies are wrong because it uses v2, not v1!

Reply:
> I'm wondering whether this should have some USE flags. I'm not sure whether sound works when SDL_mixer is disabled, possibly not. Judging by the code, it should at least have `[opengl?,video]`. I find that `[threads]` is often needed too.

SDL_Mixer is used for MIDI files, but general audio is supported by the MAD and Vorbis libraries. Although SDL_Mixer isn't necessary for sound, I have it enabled by default since it's the preferred MIDI player in PrBoom+.

I'll update the `media-libs/libsdl` depend line to add `[opengl?,joystick,video]` since those dependencies are needed in PrBoom+.

> More importantly, all your SDL dependencies are wrong because it uses v2, not v1!

Although the latest revision in the PrBoom+ subversion repository has SDL2 support, it was added after the PrBoom+ version 2.5.1.4 release, so the source tarball unfortunately contains only the SDL1 code. When the new release is made by upstream, I'll update the ebuild to depend on SDL2.

Reply:
Ah, my bad!